magento/adobe-stock-integration
# Do not show name confirmation for licensing image with saved preview

### Description (*)
This PR introduces an improvement that hides an image name prompt window in case if the image preview has been previously saved. Instead, the preview image is being replaced by the licensed image (using the name of the saved preview image).

### Fixed Issues (if relevant)
1. #866 : Image name popup should not appear when logged in user is saving licensed image if preview version was saved before
2. #874 : [MFTF] Cover Image name popup should not appear when logged in user is saving licensed image if preview version was saved before

### Manual testing scenarios (*)
1. Login to Admin Panel
2. Open Adobe Stock Integration grid from Media Gallery (i.e. Catalog -> Category -> Select Image)
3. Save preview for the image that is licensed for your account in Adobe Stock
4. Login to Adobe Stock
5. Expand preview for the image from step 3
6. Click "Save" button to save licensed version of the image

The image name prompt window will not appear. The preview image file will be replaced by the licensed image.
